modules/sdk/gradle-plugins-target-platform/src/main/java/com/liferay/gradle/plugins/target/platform/tasks/DependencyManagementTask.java

```diff
@@ -0,0 +1,135 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.gradle.plugins.target.platform.tasks;
+
+import com.google.common.collect.Maps;
+
+import com.liferay.gradle.plugins.target.platform.internal.util.GradleUtil;
+
+import groovy.util.XmlSlurper;
+import groovy.util.slurpersupport.GPathResult;
+
+import java.io.File;
+
+import java.util.Iterator;
+import java.util.Map;
+import java.util.Set;
+
+import org.gradle.api.Action;
+import org.gradle.api.DefaultTask;
+import org.gradle.api.Project;
+import org.gradle.api.artifacts.Configuration;
+import org.gradle.api.artifacts.Dependency;
+import org.gradle.api.artifacts.DependencySet;
+import org.gradle.api.logging.Logger;
+import org.gradle.api.tasks.TaskAction;
+
+/**
+ * @author Simon Jiang
+ */
+public class DependencyManagementTask extends DefaultTask {
```

Comment:
So by default this task should output to the `lifecycle` logger.  

However, we should also support a task parameter, see here:  https://docs.gradle.org/5.6.4/userguide/custom_tasks.html#sec:declaring_and_using_command_line_options

With this we can provide an `@Option` annotation and then we can have something like

```
$ gw dependencyManagement --outputFile=dm.txt
```
 and then all of the output should go to the output file.

That way from the IDE we can pass the parameter in directly and that way you can write the output more effectively.  

Actually we could also add `--outputFormat=json` or `--outputFormat=xml` and then we could write the output in a format that could be more easily read by the IDE tooling, removing the issues with text encoding/parsing.